app/javascript/pages/account/account-settings.tsx

```diff
@@ -173,8 +210,13 @@ const NameSection = ({ user, setUser }: SectionProps) => {
     handleSubmit,
   } = useForm({ mode: "all" })
 
+  const onChange = () => {
+    handleBanners("nameUpdateBanner")
+  }
+
   const onSubmit = async (data: { firstName: string; middleName: string; lastName: string }) => {
     setLoading(true)
+    handleBanners("nameSavedBanner")
```

Comment:
Should this be passed to the updateNameOrDOB function as a prop so that it can happen in the `.then` block? This way if there is an error we aren't showing the saved banner